xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -2,7 +2,7 @@
 <!-- edited with XMLSpy v2010 rel. 3 sp1 (http://www.altova.com) by Nicholas Knowles (Kizoom) -->
 <xsd:schema xmlns="http://www.netex.org.uk/netex" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:netex="http://www.netex.org.uk/netex" targetNamespace="http://www.netex.org.uk/netex" elementFormDefault="qualified" attributeFormDefault="unqualified" version="1.2.2" id="netex_ifopt_equipmentAccess_version">
 	<xsd:include schemaLocation="../../netex_framework/netex_reusableComponents/netex_facility_support.xsd"/>
-	<xsd:include schemaLocation="netex_ifopt_path_support.xsd"/>
+	<!-- <xsd:include schemaLocation="netex_ifopt_path_support.xsd"/> -->
```

Comment:
Is this really your intention?

Reply:
no.. trying to find the problem with the non-deterministic shema.